gradle/lib/dependabot/gradle/file_parser/repositories_finder.rb

```diff
@@ -36,9 +36,7 @@ def repository_urls
           repository_urls += own_buildfile_repository_urls
           repository_urls = repository_urls.uniq
 
-          return repository_urls unless repository_urls.empty?
-
-          [CENTRAL_REPO_URL]
+          return (repository_urls + [CENTRAL_REPO_URL]).uniq
```

Comment:
Also `uniq` it here. Not sure if that is necessary but felt like having multiple possibly same repos was the wrong thing to do.